android/src/play/java/com/dooboolab/RNIap/RNIapModule.kt

```diff
@@ -92,7 +92,10 @@ class RNIapModule(
             return
         }
 
-        billingClient.startConnection(
+        billingClient = BillingClient.newBuilder(reactContext).enablePendingPurchases().setListener(this)
```

Comment:
This was moved up so BillingClient can be mocked for unit tests

Reply:
I see, but if we don't create a new instance here, it's causing this error:
<img width="1039" alt="Screen Shot 2022-06-24 at 10 47 20" src="https://user-images.githubusercontent.com/16003411/175500604-50490821-f2a0-4853-9865-447c7dc6b980.png">

The rest can be reverted but I guess, we should keep this change.